api/catalog_test.go

```diff
@@ -49,7 +49,7 @@ func TestAPI_CatalogNodes(t *testing.T) {
 					"wan": "127.0.0.1",
 				},
 				Meta:        map[string]string{},
-				CreateIndex: meta.LastIndex,
+				CreateIndex: meta.LastIndex - 1,
```

Comment:
Ok for now since this is pretty reliable. We can tackle this in another PR where we will out the rest of the API tests.